components/bl-pagination/README.md

```diff
@@ -10,16 +10,16 @@ The component based on external [pagination](https://mui.com/material-ui/react-p
-| isPrevButtonVisible      | Checkbox                                                   | `true`        |                     | NO           | YES        | Allows to determine the visible or not visible "Previous" button           |
-| isFirstPageButtonVisible | Checkbox                                                   | `true`        |                     | NO           | YES        | Allows to determine the visible or not visible "First Page" button         |
-| isLastPageButtonVisible  | Checkbox                                                   | `true`        |                     | NO           | YES        | Allows to determine the visible or not visible "Last Page" button          |
+| Property                                                 | Type                                                                                                  | Default Value    | Logic               | Data Binding | UI Setting | Description                                                                |
+|----------------------------------------------------------|-------------------------------------------------------------------------------------------------------|------------------|---------------------|--------------|------------|----------------------------------------------------------------------------|
+| Page Count: `pageCount`                                  | Number                                                                                                | 10               | Page Count Logic    | YES          | YES        | Allows to determine the count of pages                                     |
+| Sibling Count: `siblingCount`                            | Number                                                                                                | 1                | Sibling Count Logic | YES          | YES        | Allows to determine how many digits to display either side of current page |
+| Variant: `variant`                                       | Select [Normal: `normal`, Outline: `outline`, Rounded: `rounded`, Rounded-Outline: `rounded-outline`] | Normal: `normal` |                     | NO           | YES        | Allows to determine the variant of pagination style                        |
+| Size: `size`                                             | Select [Small: `small`, Normal: `normal`, Large: `large`]                                             | Normal: `normal` |                     | NO           | YES        | Allows to determine the size for pagination                                |
+| Is Next Button Visible: `isNextButtonVisible`            | Checkbox                                                                                              | `true`           |                     | NO           | YES        | Allows to determine the visible or not visible "Next" button               |
```

Comment:
That should be names as setting `Is Next Button Visible` -  `Next Button`

Same for other